data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+
+	government "Deep Security"
+		"attitude toward"
+			"Deep Security that won't enter wormhole" 1.0
+	government "Deep Security that won't enter wormhole"
+		"attitude toward"
+			"Deep Security" 1.0
+
+event "hai ladybug available"
+	shipyard "Hai Intermediate"
+		"Ladybug"
+	shipyard "Hai Advanced"
+		"Ladybug"
+
+mission "Hai Reveal: Reputation Reset"
+	landing
+	invisible
+	to offer
+		has "event: pug media coverage"
+	on offer
+		# Sync reputations for aliases of governments:
+		"reputation: Deep Security that won't enter wormhole" = "reputation: Deep Security"
+		"reputation: Republic that won't enter wormhole" = "reputation: Republic"
+		"reputation: Free Worlds that won't enter wormhole" = "reputation: Free Worlds"
+		"reputation: Syndicate that won't enter wormhole" = "reputation: Syndicate"
+
+		# Make sure the new governments start with appropriate rep:
+		"reputation: Human Alliance that won't enter wormhole" = 1000
+		"reputation: Human Alliance" = 1000
+		"reputation: Hai (Friendly Unfettered)" = 1000
+		"reputation: Bounty Hunter that Attacks Hai" = -1000
+		"reputation: Philosophers" = 1000
+		"reputation: Pirate (Devil-Run Gang)" = -1000
+		fail
+
+mission "Event: Pug Media Coverage"
+	landing
+	source
+		government "Republic" "Syndicate" "Free Worlds"
+		attributes "spaceport"
+	to offer
+		has "event: pug media coverage"
+	on offer
+		event "hai secret leaks" 10 15
+		conversation
+			`It's been weeks now but as you walk through the spaceport you can't help but notice that coverage on the events of the Pug invasion still continues to dominate headlines.`
+			`	As you look around you see that experts and officials have given way to talking heads and theorists. Some of them are saying what you yourself suspect, that the Pug invasion was a ploy to cause humanity to band together, learn to resolve differences without conflict and learn not to tolerate the sorts of extremism that lead to the events that precipitated the war.`
+			`	Others...`
+			`	You sigh deeply as you watch some paranoid fringe world politician demand that the edges of human space be better understood, that threats and even potential threats must clearly exist beyond what is already known and that it was the responsibility of the government to find out and inform the people.`
+			`	Your thoughts, you keep to yourself.`
+				decline
+
+event "hai secret leaks"
+
+
+
+conversation "hai reveal free worlds news"
+	`You are relaxing at the spaceport while your ship refuels after conducting a spot of business. On the viewscreen is some talk show or other, not something you specifically paid attention to but apparently a quite popular one out of Tarazed.`
+	`	A young and fresh-faced man who looks incredibly ill at ease has just been brought out as a surprise guest by the host Jonathon Camembert.`
+	`	"So now in our surprise segment let me introduce you to Ronald. He once crewed on one of these big trade ships but after seeing what he saw, he just had to come foward!"`
```

Comment:
The use of "these" is colloqial to imply ownership. Tarazed is a major port that has a major shipyard so it's used in the sense of "*these* ships that are part of our lives", as opposed to a backwater which would say "those ships that visit occasionally".
Otherwise, good catch. 👍